docs/Makefile.custom

```diff
@@ -0,0 +1,18 @@
+# TODO(@tadeboro): Replace the shell function with the wildcard function once
+# we remove sensu_go_ prefix from all the modules (which will also mean that
+# we have all modules properly documented).
+
+modules := $(shell find ../plugins/modules/ -name '*.py' | grep -v sensu_go_)
+module_docs := $(modules:../plugins/modules/%.py=source/modules/%.rst)
+
+.PHONY: all
+docs: $(module_docs)
+	$(MAKE) html
+
+.PHONY: clean
+clean:
+	rm -rf build source/modules/*
```

Comment:
This leaves us with an empty `source/modules` directory. Which leaves me wondering, why aren't we deleting `source/modules` altogether?

```suggestion
	rm -rf build source/modules
```

Reply:
When I initially wrote this, I had no `mkdir -p source/modules` command in the documentation extraction rule. Because `ansible-doc-extractor` expects target directory to exist, this was an easy way of making sure clean does not break the build.

Of course, I changed a lot of things since then and forgot about this rule. So thanks for noticing my inconsistency.